features/netsocket/PPPInterface.cpp

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright (c) 2018 ARM Limited
```

Comment:
Please update copyrigth year in all new files

Reply:
Corrected